core/src/test/scala/unit/kafka/raft/KafkaNetworkChannelTest.scala

```diff
@@ -139,6 +145,25 @@ class KafkaNetworkChannelTest {
     }
   }
 
+  @Test
+  def testNonRoutableAddressUpdateRequest(): Unit = {
+    val destinationId = 2
+    assertThrows(classOf[IllegalArgumentException],
+      () => new InetAddressSpec(new InetSocketAddress("0.0.0.0", 0)))
```

Comment:
Can we move this to `RaftConfigTest`? It's not really part of the behavior of `KafkaNetworkChannel`.

Reply:
All the tests in `RaftConfigTest` were migrated to `KafkaConfigTest` in an earlier PR. I'll move this also.